pet-store-react-hooks/src/Home/Home.tsx

```diff
@@ -0,0 +1,73 @@
+import { useCallback, useEffect, useRef, useState } from 'react';
+
+import { getAllPetsAsync, getPetKindsAsync } from '~infrastructure/api';
+import type { PetListItem } from '~infrastructure/api-types';
+import { ErrorMessage } from '~infrastructure/components/errorMessage/ErrorMessage';
+import { LoadingIndicator } from '~infrastructure/components/loadingIndicator/LoadingIndicator';
+
+import { PetsTable } from './PetsTable';
+
+import './Home.css';
+
+export function Home() {
+  const [loading, setLoading] = useState<boolean>(false);
+  const [error, setError] = useState<string | undefined>(undefined);
```

Comment:
You can skip passing `undefined` as a default value as it will be `undefined` anyway and TS knows that.